### PR TITLE
Feature/v1.0.0

### PR DESCRIPTION
**Pull request by sajadevo**
Dashboard design for Material Tailwind

### CODE REVIEW

**Review by EINazare — CHANGES_REQUESTED** [review]
`npm install` throws a couple of warnings:

Screen Shot 2021-04-27 at 11 59 21 AM

And `npm start` does not work:

Screen Shot 2021-04-27 at 11 59 38 AM

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
+            integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w=="
+            crossorigin="anonymous"
+        />
+        <link
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+            rel="stylesheet"
+        />
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
```
[the remark this answers is not in the document]
Comment (sajadevo) [reply public/index.html]: Its the script that we need to use the material icons font

[the remark this answers is not in the document]
Comment (EINazare) [reply public/index.html]: Hello dude. I was curios as to what does 
```
<link rel="preconnect" href="https://fonts.gstatic.com" />
```
this link do?
The idea is, that in browser, I cannot navigate to it: https://fonts.gstatic.com/, it gives me a 404 page.

[the remark this answers is not in the document]
Comment (sajadevo) [reply public/index.html]: I checked it and in the new version of material icons they remove this link it was from the previous one I remove it and now we don't have anything like this: <link rel="preconnect" href="https://fonts.gstatic.com" />

**Review by groovemen — APPROVED** [review]

**Pull request closed without merging**